MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -65,18 +65,21 @@ open class BaseInstructionsBannerView: UIControl {
     }
     
     @objc func draggedInstructionsBanner(_ sender: Any) {
-        if let gestureRecognizer = sender as? UIPanGestureRecognizer, gestureRecognizer.state == .ended {
+        if let gestureRecognizer = sender as? UIPanGestureRecognizer, gestureRecognizer.state == .ended, let delegate = delegate {
             stepListIndicatorView.isHidden = !stepListIndicatorView.isHidden
-            delegate?.didDragInstructionsBanner?(self)
+            delegate.didDragInstructionsBanner?(self)
         }
     }
     
     @objc func tappedInstructionsBanner(_ sender: Any) {
-        stepListIndicatorView.isHidden = !stepListIndicatorView.isHidden
-        delegate?.didTapInstructionsBanner?(self)
+        if let delegate = delegate {
+            stepListIndicatorView.isHidden = !stepListIndicatorView.isHidden
+            delegate.didTapInstructionsBanner?(self)
+        }
     }
     
     func set(_ instruction: VisualInstructionBanner?) {
+        stepListIndicatorView.isHidden = delegate == nil
```

Comment:
Nit: Maybe some parenthesis around (delegate == nil) might make this a bit more readable? We should come up with a stylistic rule of thumb for this.